docs/source/nas/space_hub.rst

```diff
@@ -0,0 +1,155 @@
+   * - :class:`~nni.retiarii.hub.pytorch.AmoebaNet`
+     - Proposed by `Regularized evolution for image classifier architecture search <https://arxiv.org/abs/1802.01548>`__, subtly different from NASNet
+   * - :class:`~nni.retiarii.hub.pytorch.PNAS`
+     - Proposed by `Progressive neural architecture search <https://arxiv.org/abs/1712.00559>`__, subtly different from NASNet
+   * - :class:`~nni.retiarii.hub.pytorch.DARTS`
+     - Proposed by `Darts: Differentiable architecture search <https://arxiv.org/abs/1806.09055>`__, most popularly used in evaluating one-shot algorithms
+   * - :class:`~nni.retiarii.hub.pytorch.ProxylessNAS`
+     - Proposed by `ProxylessNAS <https://arxiv.org/abs/1812.00332>`__, based on MobileNetV2.
+   * - :class:`~nni.retiarii.hub.pytorch.MobileNetV3Space`
+     - The largest space in `TuNAS <https://arxiv.org/abs/2008.06120>`__.
+   * - :class:`~nni.retiarii.hub.pytorch.ShuffleNetSpace`
+     - Based on ShuffleNetV2, proposed by `Single Path One-shot <https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123610528.pdf>`__
+   * - :class:`~nni.retiarii.hub.pytorch.AutoformerSpace`
+     - Based on ViT, proposed by `Autoformer <https://arxiv.org/abs/2107.00651>`__
+
+.. note::
+
+   We are actively enriching the model space hub. Planned model spaces include:
+
+   - `NAS-BERT <https://arxiv.org/abs/2105.14444>`__
+   - `LightSpeech <https://arxiv.org/abs/2102.04040>`__
+
+   We welcome suggestions and contributions.
+
+Using pre-searched models
+-------------------------
+
+One way to use the model space is to directly leverage the searched results. Note that some of them have already been well-known neural networks and widely used.
+
+.. code-block:: python
+
+   from nni.retiarii.hub.pytorch import MobileNetV3Space
+
+   # Load one of the searched results from MobileNetV3 search space.
+   mobilenetv3 = MobileNetV3Space.load_searched_model(
+       'mobilenetv3-small-100',        # Available model alias are listed in the table below.
+       pretrained=True, download=True  # download and load the pretrained checkpoint
+   )
+
+   # MobileNetV3 model can be directly evaluated on ImageNet
+   dataset = ImageNet(directory, 'val', transform=test_transform)
+   model.eval()
+   with torch.no_grad():
+       correct = total = 0
+       for inputs, targets in pbar:
+           logits = model(inputs)
+           _, predict = torch.max(logits, 1)
+           correct += (predict == targets).sum().item()
+           total += targets.size(0)
+   print('Accuracy:', correct / total)
+
+In the example above, ``MobileNetV3Space`` can be replaced with any model spaces in the hub, and ``mobilenetv3-small-100`` can be any model alias listed below.
+
++-------------------+------------------------+----------+---------+-------------------------------+
+| Search space      | Model                  | Dataset  | Metric  | Eval Protocol                 |
```

Comment:
how about "Protocol" to "configuration" or "setting"?

Reply:
I think "protocol" is more oftenly ued in literature. I can change it to "setting" however.

Reply:
feel free to use any one you like if "protocol" is a commonly used word